pyproject.toml

```diff
@@ -17,6 +17,7 @@ pydantic = {extras = ["email"], version = "^1.8.2"}
 mch-api = {git = "https://github.com/Shaibujnr/mch-api.git", rev = "make_mch_api_installable"}
 Faker = "^10.0.0"
 python-multipart = "^0.0.5"
+climsoft-api = {git = "https://github.com/Shaibujnr/climsoft-api.git", rev = "setup-climsoft"}
```

Comment:
Can you raise a PR to get the Climsoft API code into the main `climsoft-api` repository?